dashbuilder/dashbuilder-client/dashbuilder-renderers/dashbuilder-renderer-c3/src/main/java/org/dashbuilder/renderer/c3/mutationobserver/MutationObserverInitWrapper.java

```diff
@@ -0,0 +1,20 @@
+package org.dashbuilder.renderer.c3.mutationobserver;
+
+import elemental2.dom.MutationObserverInit;
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+import jsinterop.base.Js;
+import jsinterop.base.JsPropertyMap;
+
+
+//This can be removed once Elemental is updated do RC1, where a create method will be available for MutationObserverInit
```

Comment:
@tiagobento can we update our Elemental implementation for RC1?

Reply:
@ederign I don't see any reason against it. We should open PRs on `errai`, `droolsjbpm-build-botstrap`, and `appformer` with the upgrade and wait for the full downstream build.. Since we were using a beta version of Elemental2, there might've been some breaking changes comparing to RC1.

Reply:
Created this: 
https://issues.jboss.org/browse/AF-1791

Created the following PRs: 

https://github.com/kiegroup/appformer/pull/606
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/913
https://github.com/errai/errai/pull/372

Could we merge this PR and then after AF-1791 is resolved come back and create a new ticket?

Thanks

Reply:
+1 to proceed this step by step (first merge this PR, then update elemental, then remove the workarounds needed for older elemental).